src/common/workspace-store.ts

```diff
@@ -134,10 +134,20 @@ export class WorkspaceStore extends BaseStore<WorkspaceStoreModel> {
   static readonly defaultId: WorkspaceId = "default";
   private static stateRequestChannel = "workspace:states";
 
+  @observable currentWorkspaceId = WorkspaceStore.defaultId;
+  @observable workspaces = observable.map<WorkspaceId, Workspace>();
+
   private constructor() {
     super({
       configName: "lens-workspace-store",
     });
+    const defaultWorkspace = new Workspace({
+      id: WorkspaceStore.defaultId,
+      name: "default"
+    });
+
+    defaultWorkspace.enabled = true;
+    this.workspaces.set(WorkspaceStore.defaultId, defaultWorkspace);
```

Comment:
Definitely better, though I still don't understand why the `Workspace` constructor doesn't allow setting the `enabled` flag to true.